src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseEditableBlock.tsx

```diff
@@ -65,12 +86,33 @@ const ReleaseEditableBlock = ({
 
   const handleBlur = (isDirty: boolean) => {
     if (isDirty) {
-      setUnsavedEdits(addUnsavedEdit(unsavedEdits, sectionId, block.id));
+      addUnsavedBlock(block.id);
     }
   };
 
   const handleCancel = () => {
-    setUnsavedEdits(removeUnsavedEdit(unsavedEdits, sectionId, block.id));
+    removeUnsavedBlock(block.id);
+  };
+
+  const handleSaveComment = async (comment: AddComment) => {
+    const addedComment = await releaseContentCommentService.addContentSectionComment(
```

Comment:
Redundant variable declaration, we can just return immediately

Reply:
Done